README.rst

```diff
@@ -96,6 +96,8 @@ You can pick which you prefer, but remember that these settings are handled in t
 +----------------------+--------------------------------------+------------------------------------------------------+----------------------------------------------------+------------------------------+
 | enable zen discovery | discovery_zen_ping_multicast_enabled | --elasticsearch-discovery-zen-ping-multicast-enabled | elasticsearch_discovery_zen_ping_multicast_enabled | False                        |
 +----------------------+--------------------------------------+------------------------------------------------------+----------------------------------------------------+------------------------------+
+| transport tcp port   | transport_tcp_port                   | --elasticsearch-discovery-zen-ping-multicast-enabled | elasticsearch_transport_tcp_port                   | random                       |
```

Comment:
`--elasticsearch-discovery-zen-ping-multicast-enabled` should be `--elasticsearch-transport_tcp_port`

Reply:
Plus it lacks configuration page.